### PR TITLE
Added wrapper for Base.rem (maps to remainderq) and also Base.Integer…

### LINKED ISSUE #64: Base.rem not extended to Float128
It would be nice if the following worked.
```
julia @v1.10> using Quadmath

julia @v1.10> r1, r2 = Float128(2), Float128(7);

julia @v1.10> h = (r2 - r1) / 10000;

julia @v1.10> z = [r1:h:r2;]
ERROR: rem not defined for Float128
...
```

### PR DESCRIPTION
**Pull request by macd**
…(x::Float128) = Int64(x) and a single test case that uses both. Fixes #64

### CODE REVIEW

**Comment by macd** [comment]
Well the statement that "Added line #L361 was not covered by tests" is just wrong because the test I added will throw if Base.rem does not accept Float128. So I guess the code coverage expects explicit tests?

File: src/Quadmath.jl [hunk 1]
```diff
@@ -355,6 +355,11 @@ end
     Float128(@ccall(libquadmath.hypotq(x::Cfloat128, y::Cfloat128)::Cfloat128))
 @assume_effects :foldable atan(x::Float128, y::Float128) =
     Float128(@ccall(libquadmath.atan2q(x::Cfloat128, y::Cfloat128)::Cfloat128))
+
+Base.Integer(x::Float128) = Int64(x)
```
Comment (simonbyrne) [inline src/Quadmath.jl:359]: To be consistent with Base?:
```suggestion
Base.Integer(x::Float128) = Int(x)
```

**Comment by simonbyrne** [comment]
Can you also bump the patch version?

**Comment by macd** [comment]
So the project codecov results got worse even though the patch is covered? I looked at the detailed results, but it is tough for me to see how that happened.

**Review by simonbyrne — APPROVED** [review]

**Comment by simonbyrne** [comment]
Thanks!

**Comment by macd** [comment]
And thanks to you too for the quick response!

**Pull request merged**